core/src/main/java/org/frankframework/soap/SoapWrapperPipe.java

```diff
@@ -101,6 +103,10 @@ public void configure() throws ConfigurationException {
 				setSoapNamespaceSessionKey(DEFAULT_SOAP_NAMESPACE_SESSION_KEY);
 			}
 		}
+		if (getDirection() == Direction.WRAP && PipeLine.INPUT_WRAPPER_NAME.equals(getName()) && soapVersion == SoapVersion.AUTO) {
+			ConfigurationWarnings.add(this, log, "SoapWrapperPipe should NOT be used to wrap a message in an InputWrapper, without a " +
+					"specified SoapVersion. Add a soapVersion property to the configuration. Now defaults to 1.1 without guarantees. ", SuppressKeys.CONFIGURATION_VALIDATION);
```

Comment:
Typo? 'Add a soapVersion'

Reply:
Wat is de typo dan precies? Kun je een suggestie doen?

Reply:
Weghalen van `a `, alleen "Add {@code soapVersion} property ..."

Reply:
Het is geen Javadoc, dus volgens mij wordt dat niet gesupport. Heb er effe wat tijd aan besteed om het te testen, maar kreeg het helaas niet in beeld. Heb het nu gelijk getrokken aan de manier waarop andere meldingen worden gedaan, zoals: `attribute [abc] is not supported`.